.vscode/settings.json

```diff
@@ -25,5 +25,9 @@
       }
     },
   ],
-  "java.test.defaultConfig": "WPIlibUnitTests"
+  "java.test.defaultConfig": "WPIlibUnitTests",
+  "[java]": {
+    "spotlessGradle.format.enable": true,
+    "spotlessGradle.diagnostics.enable": true
```

Comment:
Looked through the extension page, one other recommendation: maybe make it use spotless by default w/ `"editor.defaultFormatter": "richardwillis.vscode-spotless-gradle"`? The Java extension pack VS Code recommends probably also has a formatter, unless you know there's something else overriding that.

Reply:
I tested that this does use spotless when doing the Format Document command so I'm pretty sure we're good with just these settings?